Source/Core/BoxGeometry.js

```diff
@@ -34,6 +36,8 @@ define([
      * @constructor
      *
      * @param {Object} options Object with the following properties:
+     * @param {Cartesian3} options.minimum The minimum x, y, and z coordinates of the box.
+     * @param {Cartesian3} options.maximum The maximum x, y, and z coordinates of the box.
      * @param {Cartesian3} options.minimumCorner The minimum x, y, and z coordinates of the box.
```

Comment:
Remove the doc for `minimumCorner` and `maximumCorner` (since we don't want users to use them).  Same comment for `BoxOutlineGeometry`.